pilot/pkg/model/service.go

```diff
@@ -119,11 +119,25 @@ const (
 )
 
 const (
-	// MTLSReadyLabelShortname name used for determining endpoint level tls transport socket configuration
-	MTLSReadyLabelShortname = "mtlsReady"
+	// TLSModeLabelShortname name used for determining endpoint level tls transport socket configuration
+	TLSModeLabelShortname = "tlsMode"
 
-	// MTLSReadyLabelName name for the mtlsReady label given to service instances to toggle mTLS autopilot
-	MTLSReadyLabelName = "security.istio.io/" + MTLSReadyLabelShortname
+	// TLSModeLabelName name for the mtlsReady label given to service instances to toggle mTLS autopilot
```

Comment:
comment is no longer accurate. `mTLS autopilot` now you use this label for other cases more than mTLS..